testing/test_regressions.py

```diff
@@ -35,7 +35,8 @@ def test_pkginfo_noscmroot(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> N
 
     tmp_path.joinpath(".git").mkdir()
     p.joinpath("setup.py").write_text(
-        "from setuptools import setup;" 'setup(use_scm_version={"root": ".."})',
+        '''from setuptools import setup;
```

Comment:
Do we want to keep `setup.py` this as an one-liner?
```python
from setuptools import setup;setup(use_scm_version={"root": ".."})
```
Or perhaps change as follows?
```python
from setuptools import setup
setup(use_scm_version={"root": ".."})
```